compiler/src/main/scala/dx/translator/wdl/CallableTranslator.scala

```diff
@@ -526,6 +527,8 @@ case class CallableTranslator(wdlBundle: WdlBundle,
                 |""".stripMargin
         )
         val blockName = s"${wfName}_block_${pathStr}"
+
+        // TODO block subworkflow is created here
```

Comment:
Please update the comment to sth like: A block subworkflow is created here

Reply:
Fixed in [9c3ecae](https://github.com/dnanexus/dxCompiler/pull/291/commits/9c3ecae9726a388bdd2185c61bc605de4f5f251f)